tests/integration/README.md

```diff
@@ -0,0 +1,386 @@
+echo -e "user = \"$(whoami)\"\ngroup = \"$(whoami)\"" | sudo tee -a /etc/libvirt/qemu.conf
+
+# apply changes
+sudo systemctl restart libvirtd
+```
+
+2. Create and setup VMs
+```bash
+# cd to the c2r-vagrant repo dir, i.e.
+cd ~/Documents/c2r-vagrant
+
+# Build vagrant VM
+SYSTEM=centos8 SYSTEM_SESSION=1 vagrant up
+
+# [OPTIONAL] generate your ssh key if you still don't have one (ls -al ~/.ssh/*.pub)
+# follow instructions in https://docs.gitlab.com/ee/ssh/#generate-an-ssh-key-pair
+
+# copy you ssh key to the VM (you'll be asking to enter the password (`vagrant`)
+ssh-copy-id root@$(virsh domifaddr --domain c2r-vagrant_default | gawk 'match($0, /(192.+)\/.+/, ary) {print ary[1]}')
+
+# ssh to the vagrant VM
+ssh root@$(virsh domifaddr --domain c2r-vagrant_default | gawk 'match($0, /(192.+)\/.+/, ary) {print ary[1]}')
+    # update packages
+    yum update -y
+    
+    # Disable blacklisted rpc in qemu config
+    vim /etc/sysconfig/qemu-ga
+    # comment line starting with BLACKLIST_RPC and save (:wa)
+    #
+    
+    # Switch selinux into permissive mode
+    vim /etc/selinux/config
+    # set SELINUX=permissive as save (:wa)
+    #
+    
+    # enable qemu-ga
+    systemctl enable qemu-guest-agent
+    
+    # install python pip
+    # for centos8 and ol8
+    curl https://bootstrap.pypa.io/pip/get-pip.py | python3
+    # for centos7 and ol7
+    # curl https://bootstrap.pypa.io/pip/2.7/get-pip.py | python
+    
+    # install rsync
+    yum install -y rsync
+    
+    # Remove installed version of convert2rhel
+    yum remove convert2rhel -y
+    
+    # exit the VM Ctrl+D
+
+# shutdown the vagrant VM
+virsh shutdown c2r-vagrant_default
+
+# clone the VM and give it corresponding name (--name)
+# SYSTEM=centos8 -> c2r_centos8_template
+# SYSTEM=centos7 -> c2r_centos7_template
+# SYSTEM=ol8 -> c2r_oracle8_template
+# SYSTEM=ol7 -> c2r_oracle7_template
+virt-clone --original c2r-vagrant_default --name c2r_centos8_template --auto-clone --check disk_size=off
+
+# remove the origin vagrant VM
+virsh undefine --domain c2r-vagrant_default
+
+# Now do the same for SYSTEM={centos7,ol8,ol7}
+```
+
+3. Perform post setup for each VM 
+   
+3.1. In Virtual Machine Manager go to Edit -> Preferences and set enable
+xml editing
+
+3.2. For each VM in VM Manager click Open and:
+- set network source (NIC device) to default (Virtual network 'default': NAT)
+- add qemu-ga channel, by clicking Add Hardware button:
+select Channel, use XML to declar the device and add:
+```xml
+<channel type="unix">
+  <target type="virtio" name="org.qemu.guest_agent.0"/>
+  <address type="virtio-serial" controller="0" bus="0" port="1"/>
+</channel>
+```
+click Finish, click Apply
+
+## Step 3 - Setup TMT-Pytest framework
+Install local development environment:
+```bash
+# cd to you convert2rhel dir
+cd ~/Documents/convert2rhel/
+
+# install venv, python deps and build images
+rm .images .install
+make install
+
+# set secrets for running integration tests
```

Comment:
```suggestion
# set RHSM-related variables for running integration tests
```

Reply:
Solved it a bit differently - added this note in .env.example:
```
# RHSM (Red Hat Subscription Manager related secrets)
RHSM_POOL=
RHSM_PASSWORD=
RHSM_USERNAME=
RHSM_SERVER_URL=
```
so we'll have space for adding another secrets